clients/spark/core/src/main/scala/io/treeverse/clients/LakeFSContext.scala

```diff
@@ -15,8 +15,8 @@ object LakeFSContext {
   val LAKEFS_CONF_API_URL_KEY = "lakefs.api.url"
   val LAKEFS_CONF_API_ACCESS_KEY_KEY = "lakefs.api.access_key"
   val LAKEFS_CONF_API_SECRET_KEY_KEY = "lakefs.api.secret_key"
-  val LAKEFS_CONF_API_CONNECTION_TIMEOUT_KEY = "lakefs.api.connection.timeout"
-  val LAKEFS_CONF_API_READ_TIMEOUT_KEY = "lakefs.api.read.timeout"
+  val LAKEFS_CONF_API_CONNECTION_TIMEOUT_SEC_KEY = "lakefs.api.connection.timeout.seconds"
+  val LAKEFS_CONF_API_READ_TIMEOUT_SEC_KEY = "lakefs.api.read.timeout.seconds"
```

Comment:
```suggestion
  val LAKEFS_CONF_API_CONNECTION_TIMEOUT_SEC_KEY = "lakefs.api.connection.timeout_seconds"
  val LAKEFS_CONF_API_READ_TIMEOUT_SEC_KEY = "lakefs.api.read.timeout_seconds"
```

We'll end up using `lakefs.api.connection.timeout` and `lakefs.api.read.timeout`, and having something that looks like a field in them will be confusing.

Reply:
done